src/main/java/net/imglib2/mesh/alg/MeshConnectedComponents.java

```diff
@@ -36,7 +36,7 @@ public static final int n( final Mesh mesh )
 	{
 		final Triangles triangles = mesh.triangles();
 		final Vertices vertices = mesh.vertices();
-		final int nVertices = ( int ) vertices.size();
+		final int nVertices = ( int ) vertices.sizel();
```

Comment:
Would be simpler to write:
```java
final int nVertices = verticies.size();
```
rather than calling `sizel()` and then casting down.

Reply:
Done!